src/main.rs

```diff
@@ -1669,6 +1677,8 @@ impl Application for App {
                         let term = terminal.term.lock();
                         if let Some(text) = term.selection_to_string() {
                             return Command::batch([clipboard::write(text), self.update_focus()]);
+                        } else {
+                            println!("Copied, but no text selection!");
```

Comment:
At this point how can a `^C` signal be sent to the terminal?

Reply:
Sending 0x03 with the input_scroll function.